_extras/snippets_library/EPCC_Cirrus_pbs/13/statu_output.snip

```diff
@@ -1,6 +1,11 @@
-
+{: .bash}
```

Comment:
This `{:bash}` tag belongs to the preceding block. Please remove from this snippet.